core/javatests/com/google/census/CensusScopeTest.java

```diff
@@ -41,58 +41,76 @@ public void testNoScope() {
 
   @Test
   public void testScope() {
-    try (CensusScope scope = new CensusScope(Census.getCurrent().with(K1, V1))) {
+    CensusScope scope = new CensusScope(Census.getCurrent().with(K1, V1));
```

Comment:
not sure if bazel does this, but not really required to limit tests to java6 is it?